README.md

```diff
@@ -1,21 +1,27 @@
 # FunPro
```

Comment:
There is a typo at `No how do I use the values when` . Should be `Now how do I use the values when`.
I couldn't add a comment on the line since there is no change there.